Assets/_Scripts/Tile.cs

```diff
@@ -5,11 +5,11 @@
 
 public class Tile : MonoBehaviour
 {
-    private List<Vector3> validDirections = new();
+    private List<Vector3> validDirections = new List<Vector3>();
     private Vector3[] checklist;
     private float destroyTime = 0.3f;
 
-    private void Start()
+    private void Awake()
```

Comment:
Not that I'm asking this be changed, but why awake and not start?

Reply:
Check List gives a null reference when in start becuase player controller calls GetValidDirections() one of the first thing in the project but CheckList is null at that specific frame. So i moved it to awake and it doesnt give those errors anymore